### PR TITLE
Remove doc about function that is not being only a proxy

### PR DESCRIPTION
**Pull request by joaothallis**
For `insert_all/2` it is creating a span: https://github.com/open-telemetry/opentelemetry-erlang-contrib/blob/8af382e26a8bbf306662bc745f1c000ad1323cc5/instrumentation/opentelemetry_oban/lib/opentelemetry_oban.ex#L107

### CODE REVIEW

**Review by bryannaegele — COMMENTED** [review]
This diff shows there are no content changes, just formatting changes. Did something get left out?

**Comment by bryannaegele** [comment]
Ok. There are several grammatical errors. Can you please fix those?

**Comment by joaothallis** [comment]
> Ok. There are several grammatical errors. Can you please fix those?

Can you point some errors? I did not find it in the `README.md`

File: instrumentation/opentelemetry_oban/README.md [hunk 1]
```diff
@@ -49,6 +49,6 @@ After:
   |> OpentelemetryOban.insert()
 ```
 
-Oban also supports inserting jobs using `Oban.insert/4`, `Oban.insert_all/2`
-and `Oban.insert_all/4`. These are currently not supported by OpentelemetryOban
-and are just proxied through to Oban.
+Oban also supports inserting jobs using `Oban.insert/4`, and 
+`Oban.insert_all/4`. These are currently not supported by OpentelemetryOban and
+are just proxied through to Oban.
```
Comment (bryannaegele) [inline instrumentation/opentelemetry_oban/README.md:54]:
```suggestion
Oban also supports inserting jobs using `Oban.insert/4` and `Oban.insert_all/4`.
These are currently not supported by OpentelemetryOban and are just proxied through to Oban.
```

**Review by bryannaegele — APPROVED** [review]

**Comment by bryannaegele** [comment]
No worries, I got it :)

**Pull request merged**